### ISSUE TITLE
Docs: Link old pages to new ones

### ISSUE THREAD

**Issue by asummers1** [twbs/bootstrap#39212]
### Prerequisites

- [X] I have [searched](https://github.com/twbs/bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed feature requests
- [X] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)

### Proposal

I would like to suggest that docs for older versions of Bootstrap include links to the newest versions.

### Motivation and context

This is a fast and convenient way for users to be able to access the latest versions of docs. It is especially useful when trying to search for a topic via a search engine, where users are often directed to articles for older Bootstrap versions. If their framework's major version number is not listed in the dropdown, they must either specify the desired version number in the search term, or navigate to getbootstrap.com's homepage and search there. This is unintuitive.

**Comment by Rjchauhan18** [comment]
I would like to do this issue .
Please assign it to me.

**Comment by bluepuma77** [comment]
Agree. When you google something, you might end up on [5.2 doc](https://getbootstrap.com/docs/5.2/getting-started/introduction/). And it states "Latest (5.2.x)".

I would expect that the old documentation tells me that currently 5.3 is the latest and directly links to it.
